cypress/e2e/article.cy.js

```diff
@@ -1,24 +1,81 @@
 /// <reference types='cypress' />
 /// <reference types='../support' />
 
-describe('Article', () => {
-  before(() => {
+import ArticlePageObject from '../support/pages/article.pageObject';
+import HomePageObject from '../support/pages/home.pageObject';
+import SignInPageObject from '../support/pages/signIn.pageObject';
 
-  });
+const articlePage = new ArticlePageObject();
+const homePage = new HomePageObject();
+const signInPage = new SignInPageObject();
 
-  beforeEach(() => {
+describe('Article', () => {
+  let user;
+  let article;
+
+  before(() => {
     cy.task('db:clear');
+    cy.task('generateUser').then((generateUser) => {
+      user = generateUser;
+    });
+    cy.task('generateArticle').then((generateArticle) => {
+      article = generateArticle;
+    });
   });
 
   it('should be created using New Article form', () => {
+    signInPage.visit();
+    cy.register(user.email, user.username, user.password);
+
+    signInPage.typeEmail(user.email);
+    signInPage.typePassword(user.password);
+    signInPage.clickSignInBtn();
 
+    homePage.assertHeaderContainUsername(user.username);
+
+    articlePage.clickArticleLink();
+    articlePage.inputTitle(article.title);
+    articlePage.inputTopic(article.description);
+    articlePage.inputBody(article.body);
+    articlePage.inputTags(article.tag);
+    articlePage.clickPublishBtn();
+    articlePage.checkCreatedArticle(article.title);
   });
 
   it('should be edited using Edit button', () => {
```

Comment:
for edit and delete tests the best option is to create article in preconditions via API and use it